tests/sentry/api/endpoints/test_organization_ddm_meta.py

```diff
@@ -384,6 +384,74 @@ def test_get_metric_spans(self):
             {"spanDuration": 2, "spanOp": "rpc"},
         ]
 
+    def test_get_metric_spans_with_environment(self):
+        mri = "g:custom/page_load@millisecond"
+
+        self.create_environment(project=self.project, name="prod")
```

Comment:
We have to create the environment in db, since this is checked when the environment is passed via the endpoint.